### DISCUSSION TITLE
Regrets for new user question - Enlarging font image

### DISCUSSION THREAD

**Discussion by klsparks** [fontforge/fontforge#5172] [Q&A] [answered]
I'm new to fontforge and to font creation in general. I've created a series of characters in Adobe Ill and have successfully imported them and added them to my new font. BUT ...

While I'm able to edit and reshape the results for each character in the edit window, I cannot, for the life of me, figure out how to select the entire character and resize it. I've read the instructions. Looks like there might be a way in the tools menu, but ... another question ... the tool menu is greyed out.

So, I guess two questions: 
1. Why is tools greyed out? 
2. How do you select a character and drag it symmetrically to a larger size? 

Thanks!

**Comment by ctrlcctrlv** [comment, marked as answer]
No need to have regrets. :-)

1. Ctrl-A, or drag to select all points, or Edit→Select All
2. Use the tool with this icon ![image](https://user-images.githubusercontent.com/838783/210156945-c7267d67-59cf-48e3-b67a-1b5bc1fc4e79.png), or use Element→Transformations→Transform for more precision.

Happy New Year!

**Reply by ctrlcctrlv** [reply]
Oh and Tools is greyed out because it's for Python scripts. If you've none installed there's none to list thus grey.

**Comment by klsparks** [comment]
Thx a ton, Frederick!

Happy New Year!

On Sat, Dec 31, 2022 at 5:48 PM Fredrick Brennan ***@***.***>
wrote:

> Oh and Tools is greyed out because it's for Python scripts. If you've none
> installed there's none to list thus grey.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fontforge/fontforge/discussions/5172#discussioncomment-4562319>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5AYQVTMDFGOBLQ3ELHHCF3WQCZ3LANCNFSM6AAAAAATNYGRDA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Sent from Gmail Mobile
